src/control/system/member_test.go

```diff
@@ -1,10 +1,10 @@
 //
-// (C) Copyright 2020-2021 Intel Corporation.
+// (C) Copyright 2020-2022 Intel Corporation.
 //
 // SPDX-License-Identifier: BSD-2-Clause-Patent
 //
 
-package system
+package system_test
```

Comment:
Specifically with the goal of testing from outside of the package?

Reply:
Yes. We should be doing this as a general rule, IMO, unless we're testing unexported internal functions.

Reply:
Part of the reason we were testing from within the packages was that, and also to be able to twiddle with the unexported internals of some structs that might affect behavior. In some cases it doesn't make a difference, sure, but seems like we should establish a convention/strategy for when to use one or the other (and how to name the files in the case that we use both).

Reply:
Sure, I didn't mean to imply that it's wrong to do internal testing -- I just meant that I think we should probably do more testing from the outside to make us think a bit more about the exported surface of our packages.